src/main/java/com/google/api/codegen/gapic/GapicGeneratorApp.java

```diff
@@ -137,6 +149,22 @@ protected void process() throws Exception {
       }
     }
 
+    // Read the sample configs, if they are given, and convert them to protos.
+    SampleConfigProto sampleConfigProto = null;
+    List<String> sampleConfigFileNames = options.get(SAMPLE_CONFIG_FILES);
+    if (sampleConfigFileNames.size() > 0) {
+      ConfigSource configSource =
+          loadConfigFromFiles(
+              sampleConfigFileNames,
+              SampleConfigProto.getDescriptor().getFullName(),
+              SampleConfigProto.getDefaultInstance());
+
+      // TODO(hzyi): Verify this works for repeated fields as well
```

Comment:
Clarify?

Reply:
YamlReader will merge multiple config yamls as long as they all have the same `type` and `version` specified. but I am not sure how it handles repeated fields.